problemsets-mat223/modules/module1-exercises.tex

```diff
@@ -126,6 +126,52 @@
     		    \item Yes.
 		    \end{enumerate}
 		\end{solution}
+		
+		\prob
+		    Use set notations, vectors, or linear combinations of vectors to describe the following sets.
+		    \begin{enumerate}
+		        \item The set of vectors with rational components in $\R^2$.
+		        \item The set of vectors with irrational components in $\R^2$.
+		        \item The set that contains the empty set.
+		        \item $\R^3$.
+		    \end{enumerate}
+    		
 		% Challenge (3 questions)
+		\prob 
+		% Propose: get students to understand quantifiers and set operations
+		% practice providing justification to their intuitive guesses
+		Which of the following statements is true about the set listed below? Justify your answers.
+		    \begin{enumerate}
+		        \item The y-axis in $\R^3$.
+		        \begin{enumerate}
+		            \item It can be written as a set of linear combinations of vectors.
+		            \item There exists sets $A , B \subseteq \R^3$ such that this set is equal to $A \cup B$.
+		            \item There exists sets $A , B \subseteq \R^3$ such that this set is equal to $A \cap B$.
```

Comment:
These are always true. Can you come up with something that's not always true? Otherwise, it belongs as part of a question: "Determine whether the following statements are Always True, Sometimes True, or Never True"